server/container/guice/jpa-guice/src/main/java/org/apache/james/JPAJamesServerMain.java

```diff
@@ -95,10 +101,15 @@
     private static final Module JPA_MODULE_AGGREGATE = Modules.combine(JPA_SERVER_MODULE, PROTOCOLS);
 
     public static void main(String[] args) throws Exception {
+        Optional.ofNullable(System.getProperty("logback.configurationFile"))
+            .stream()
+            .forEach(file -> System.err.println("Load logging config from " + file));
+
         Configuration configuration = Configuration.builder()
             .useWorkingDirectoryEnvProperty()
             .build();
 
+        LOGGER.info("Loading configuration {}", configuration);
```

Comment:
Shouldn't this be maybe implemented then as well for other profiles? `CassandraRabbitMQJamesServerMain`, `CassandraJamesServerMain`, `CassandraLdapJamesServerMain` ?

Or could that be maybe generalized in `JamesServerMain`?

Reply:
It could be, but I'm not familiar with those.

Reply:
> Shouldn't this be maybe implemented then as well for other profiles? CassandraRabbitMQJamesServerMain, CassandraJamesServerMain, CassandraLdapJamesServerMain ?

:+1: that would be helpful for other servers as well.

> Or could that be maybe generalized in JamesServerMain?

:+1: again for being generic and avoiding to repeat ourselves.

JamesServerMain is just a fin layer for running a GuiceJamesServer, it is no longer aware of configuration details.

Maybe a more generic place could be `GuiceJamesServer::forConfiguration` method?

Reply:
we generally avoid implicit toString() call, could you please add it?

Reply:
I dropped the logback logging thing.
I moved some things to JamesServerMain. 
I can't do more since they are `static`s. 

It's good enough IMO.